DEVELOPMENT.md

```diff
@@ -45,6 +45,7 @@ In this case, we duplicate the YAML error type, leave one of those for auto conv
 **Test your changes**
 
 * [ ] run `cargo test` on the root to test Loco itself
+* [ ] Ensure you have the necessary local resources, such as `DB`/`Redis`, by executing the command `cargo loco doctor`. In case you don't have them, refer to the relevant documentation section for guidance.
```

Comment:
Thank. last changes:

1. Can you bump this bullet to be first? we should validate the resources before running the test.
2. add --environment test to `cargo loco doctor`. we need to check the connectivity via testing and not the default (which is development) 
should be `cargo loco doctor --environment test`

Reply:
Changed.